backend/src/plugins/ContextMenus/utils/loadAllCommands.ts

```diff
@@ -1,22 +1,25 @@
 import { ApplicationCommandData } from "discord.js";
-import { LogType } from "src/data/LogType";
-import { LogsPlugin } from "src/plugins/Logs/LogsPlugin";
+import { LogType } from "../../../data/LogType";
+import { LogsPlugin } from "../../../plugins/Logs/LogsPlugin";
 import { GuildPluginData } from "knub";
-import { ContextMenuPluginType, ContextMenuTypeNameToNumber } from "../types";
+import { ContextMenuPluginType } from "../types";
+import { hardcodedContext } from "./hardcodedContextOptions";
 
 export async function loadAllCommands(pluginData: GuildPluginData<ContextMenuPluginType>) {
   const comms = await pluginData.client.application!.commands;
-  const actions = pluginData.config.get().context_actions;
+  const cfg = pluginData.config.get();
   const newCommands: ApplicationCommandData[] = [];
   const addedNames: string[] = [];
 
-  for (const [name, configAction] of Object.entries(actions)) {
-    if (!configAction.enabled) continue;
+  for (const [name, label] of Object.entries(hardcodedContext)) {
+    if (!cfg[name]) continue;
 
+    const type = name.startsWith("user") ? 2 : 3;
```

Comment:
Perhaps use the enum from the discord.js Constants called `ApplicationCommandTypes` instead of magic numbers

Reply:
Sadly whenever I tried that I got a module not found error and when I mirror the enum I get a `xyz is not assignable to zyx` error so I decided not to bother. I'll add comments though